CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - PR #6847 Add a cmake find module for cuFile in JNI code
 - PR #6902 Implement `DataFrame.quantile` for `datetime` and `timedelta` data types
+- PR #6940 Add JNI wrapper for the cuFile API (GDS)
```

Comment:
This is not needed in 0.18 any more.  They changed how the changelog is maintained and now it is auto generated.